gateway/api/serializers.py

```diff
@@ -116,7 +116,7 @@ class RunExistingProgramSerializer(serializers.Serializer):
 
     title = serializers.CharField(max_length=255)
     arguments = serializers.CharField()
-    config = serializers.CharField()
+    config = serializers.DictField()
```

Comment:
Curious to know, why did you decide to use `DictField` instead of `JSONField` here, Aki?

Reply:
The `config` field of the `Job` is DictField so I changed the data format of the `config` sent by the client to Dict instead of str.

Reply:
Yes, the change I think is correct. But I was just curious about the `DictField` instead to use `JSONField`:

https://stackoverflow.com/questions/76178706/django-serializer-dictfield-vs-jsonfield

Reply:
I haven't thought about it. I'll test it.

Reply:
It worked by changing the post request with the Context-type to json.